src/Core/Services/StateMigrationService.cs

```diff
@@ -837,6 +841,43 @@ internal enum UsernameEmailType
 
         #endregion
 
+        #region v6 to v7 Migration
+
+        private class V7Keys
+        {
+            // global keys
+            internal const string StateKey = "state";
+            internal const string RegionEnvironmentKey = "regionEnvironment";
+            internal const string PreAuthEnvironmentUrlsKey = "preAuthEnvironmentUrls";
+        }
+
+        // Migrate environment data to use Regions
```

Comment:
Comment should be inside `MigrateFrom6To7Async` block since additional (unrelated) migrations could be added before going to production